StringDictionary.vi.xaml

```diff
@@ -88,7 +88,8 @@
 	<system:String x:Key="Messages.Upgrade.NewVersion">Bạn có thể nâng cấp lên phiên bản {0}</system:String>
 	<system:String x:Key="Messages.Upgrade.Uptodate">Bạn đang sử dụng phiên bản mới nhất</system:String>
 	<system:String x:Key="Messages.VolumeInvalidChar">Tên không thể chứa những ký tự sau:</system:String>
-	<system:String x:Key="Messages.VolumeInvalidStr">Không sử dụng các tên sau:</system:String>
+	<system:String x:Key="Messages.VolumeInvalidStr">Không sử dụng các tên sau:
+	<system:String x:Key="Messages.PasswordRequiredIncludeCredentials">Cần có mật khẩu để bao gồm thông tin xác thực.</system:String>
```

Comment:
Incorrect line is added

Reply:
> Incorrect line is added

I have edited it again

Reply:
Line#92 <system:String x:Key="Messages.PasswordRequiredIncludeCredentials">Cần có mật khẩu để bao gồm thông tin xác thực.</system:String> is overlaps with line# 69 	<system:String x:Key="Messages.PasswordRequiredIncludeCredentials">Cần phải có mật khẩu để bao gồm thông tin xác thực.</system:String>
It should be modified the content of line 69 without adding line 92.